.github/workflows/platformDeploy.yml

```diff
@@ -101,19 +94,28 @@ jobs:
           name: android-sourcemap-${{ github.ref_name }}
           path: android/app/build/generated/sourcemaps/react/productionRelease/index.android.bundle.map
 
-      - name: Upload Android version to GitHub artifacts
+      - name: Upload Android build to GitHub artifacts
         if: ${{ !fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
         uses: actions/upload-artifact@v4
         with:
           name: app-production-release.aab
           path: android/app/build/outputs/bundle/productionRelease/app-production-release.aab
 
-      - name: Upload Android version to Browser Stack
+      - name: Upload Android build to Browser Stack
         if: ${{ !fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
         run: curl -u "$BROWSERSTACK" -X POST "https://api-cloud.browserstack.com/app-live/upload" -F "file=@./android/app/build/outputs/bundle/productionRelease/app-production-release.aab"
         env:
           BROWSERSTACK: ${{ secrets.BROWSERSTACK }}
 
+      - name: Upload Android build to GitHub Release
+        if: ${{ fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
+        run: |
+          RUN_ID="$(gh run list --workflow platformDeploy.yml --event push --branch ${{ github.event.release.tag_name }} --json databaseId --jq '.[0].databaseId')"
```

Comment:
Are the `--json` and `--jq` params not conflicting with each other? I checked [these docs](https://cli.github.com/manual/gh_run_list) out and they say:

> `-q`, `--jq <expression>`
> Filter JSON output using a jq expression
>
> `--json <fields>`
> Output JSON with the specified fields

I guess `jq` is used first, but does that mean the initial output has a nested `databaseId` object `[ {"databaseId": {"databaseId": "something" }}, { ... } ]`?

Reply:
`--json` controls which fields are included in the JSON output, `--jq` filters and manipulates that json. This can be easily tested locally:

- without jq:

<img width="279" alt="image" src="https://github.com/user-attachments/assets/2ca030fd-c7b4-4443-ac35-a894b62d44f6">

- with jq:

<img width="96" alt="image" src="https://github.com/user-attachments/assets/6afd7b95-58cb-425e-a3df-1b0494137a5f">